_episodes/11-software-project.md

```diff
@@ -55,67 +55,59 @@ to fix and build on top of the existing code during the course.
 ## Downloading Our Software Project
 
 To start working on the project, you will first create a fork of the software project repository
-from GitHub within your own GitHub account
-and then obtain a local copy of that project (from your GitHub) on your machine.
+from GitLab within the workshop's GitLab group
+and then obtain a local copy of that project (from your GitLab) on your machine.
 
-1. Make sure you have a GitHub account
-   and that you have [set up SSH key pair for authentication with GitHub](../setup.html#secure-access-to-github-using-git-from-command-line).
+1. Make sure you have a GitLab account
+   and that you have [set up SSH key pair for authentication with GitLab](../setup.html#secure-access-to-gitlab-using-git-from-command-line).
```

Comment:
To check: does `setup.html` need to be updated too?

Reply:
Yes! It is on my to do list